extensions/shared/is-wpcom-site.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * External Dependencies
+ */
+import { get } from 'lodash';
+
+export default function isWpcomSite() {
+	return (
+		get( 'object' === typeof window ? window : null, [ '_currentSiteType' ], null ) === 'simple'
```

Comment:
I think the guarding here is a bit overzealous 😅 
```suggestion
		'object' === typeof window ? window._currentSiteType === 'simple' : false
```

Reply:
Haha, right on! This was a quickie copy paste from `extensions/shared/get-jetpack-data.js` without much thought that it certainly would've deserved. Thanks!